float32[] Temperature # temperature meassured at the power stage of the motor controller
float32[] Voltage # meassured input voltage of the motor controllers
int32 err_cnt # counts errors at motor
int32 to_cnt # counts timeouts! 
int32 cycl_vl # cycle violations
